src/components/fields/NumericOrDate.js

```diff
@@ -0,0 +1,38 @@
+import Field from './Field';
+import {UnconnectedNumeric} from './Numeric';
+import {UnconnectedText} from './Text';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {connectToContainer} from 'lib';
+import {isDateTime} from 'plotly.js/src/lib';
+import {isJSDate} from 'plotly.js/src/lib/dates';
+
+export class UnconnectedNumericOrDate extends Component {
+  render() {
+    const fullValueIsDate =
+      typeof this.props.fullValue === 'string' &&
+      (isDateTime(this.props.fullValue) || isJSDate(this.props.fullValue));
+
+    return fullValueIsDate ? (
+      <UnconnectedText {...this.props} placeholder={'yyyy-mm-dd 00:00:00.00'} />
+    ) : (
+      <UnconnectedNumeric {...this.props} />
+    );
+  }
+}
+
+UnconnectedNumericOrDate.propTypes = {
+  defaultValue: PropTypes.any,
+  fullValue: PropTypes.any,
+  min: PropTypes.number,
+  max: PropTypes.number,
+  multiValued: PropTypes.bool,
+  hideArrows: PropTypes.bool,
+  showSlider: PropTypes.bool,
+  step: PropTypes.number,
+  fullContainer: PropTypes.object,
+  updatePlot: PropTypes.func,
+  ...Field.propTypes,
+};
+
+export default connectToContainer(UnconnectedNumericOrDate);
```

Comment:
here's the other definition

Reply:
oups.. this is the one I don't want.. thanks!

Reply:
wait! this is the one you want